sdk/storage/blobs/inc/az_storage_blobs.h

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef AZ_STORAGE_BLOBS_H
+#define AZ_STORAGE_BLOBS_H
+
+#include <az_contract.h>
+#include <az_http_pipeline.h>
+#include <az_http_response.h>
+#include <az_result.h>
+#include <az_span.h>
+#include <az_str.h>
+
+#include <stddef.h>
+
+#include <_az_cfg_prefix.h>
+
+
+AZ_NODISCARD AZ_INLINE az_span az_storage_blobs_blob_header_blob_type() { return AZ_STR("x-ms-blob-type"); }
+AZ_NODISCARD AZ_INLINE az_span az_storage_blobs_blob_header_Content_Length() { return AZ_STR("Content_Length"); }
+
+AZ_NODISCARD AZ_INLINE az_span az_storage_blobs_blob_type_AppendBlob() { return AZ_STR("AppendBlob");} /* UnSupported */
+AZ_NODISCARD AZ_INLINE az_span az_storage_blobs_blob_type_BlockBlob() { return AZ_STR("BlockBlob");}
+AZ_NODISCARD AZ_INLINE az_span az_storage_blobs_blob_type_PageBlob() { return AZ_STR("PageBlob"); } /* UnSupported */
```

Comment:
I wanted to chat with the team, and you, and @vhvb1989 as well. So, is it official - our naming convention is to mimic the REST enum names, including the case, so it it not `page_blob`, but `PageBlob`, even though we use snake casing for the rest of the API? The same goes for `Content_Length` instead of `content_length`, some keyvault  names (@vhvb1989), and so on?

Reply:
I am fine with any name :)

Reply:
@sergey-shandar just confirmed offline that we should use snake casing. If you agree, can you please also update `az_keyvault_key_operation_unwrapKey`'s name in the keyvault module?

Reply:
No its not official.  I planned to bring these up in our Friday meeting (which has now been postponed).

Reply:
@gilbertw I brought this several times already. For now, we agree to use `snake_case` everywhere. I would like to raise this question again in Jan 2020.

Reply:
Agreed, I switched to snake_case for now.  We can discuss in January